frontend/apps/reader/modules/readerlink.lua

```diff
@@ -685,13 +685,13 @@ function ReaderLink:onGoToExternalLink(link_url)
 end
 
 --- Goes back to previous location.
-function ReaderLink:onGoBackLink(show_notification_if_empty)
+function ReaderLink:onGoBackLink(dont_show_notification_if_empty)
```

Comment:
Why was that needed? Because you can't pass arguments when the events are sent by profile? (So we should build our future onSomething() that take argument to have the more sensible defaults when used as profile?)
May be just a shorter `no_notification_if_empty` for the name.

Reply:
I ended up having a separate category that passes a argument